README.md

```diff
@@ -1 +1,143 @@
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio - Mobile Section] <a name="about-project"></a>
+
+**[Portfolio - Mobile Section]** is a great website tool that helps to secure a job as a web developer,
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML, CSS</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Github flow</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>VScode</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Install a code editor
+
+### Setup
+
+Clone this repository to your desired folder:
+
+cd my-folder
+
+git clone repo-link
+
+### Install
+
+Install this project with:
+
+### Usage
+
+To run the project, execute the following command:
+
+Press ctrl+shif+p on vscode and choose run
+
+### Run tests
+
+To run tests, run the following command:
+
+Press ctrl+shift+p on vscode and choose test
+
+### Deployment
+
+You can deploy this project using:
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="Michael"></a>
+
+👤 **Author1**
+
+
+- GitHub: [@githubhandle](https://github.com/Mike47ip)
+- Twitter: [@twitterhandle](https://twitter.com/Mikepee47)
+- LinkedIn: [@LinkedIn](https://www.linkedin.com/in/michael-darkwah-81a039141/)
+- Facebook: [@facebook](https://web.facebook.com/profile.php?id=100089565852279)
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[My Projects]**
+- [ ] **[About Me]**
+- [ ] **[Contact]**
+
+
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project show your love by leaving a comment
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank all Micronauts for their support
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/Kwaku28/Portfolio-Mobile/blob/toolbar-headline-section/MIT.md) licensed.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly 😃 as stated by the previous reviewer try to add the section of contributing to the Readme as you see here it is not existing here in your Readme, so please try to add it to make your Readme professional 💯🚀